Documentation/concepts/networking/ipam/eni.rst

```diff
@@ -405,7 +405,7 @@ ENI Deletion Policy
 ENIs can be marked for deletion when the EC2 instance to which the ENI is
 attached to is terminated. In order to enable this, the option
 ``spec.eni.delete-on-termination`` can be enabled. If enabled, the ENI
-is modifying after creation using ``ModifyNetworkInterface`` to specify this
+is modifying after creation using ``ModifyNetworkInterfaceAttribute`` to specify this
```

Comment:
Could we also fix the typo on the same line, please?
Looks good otherwise, thanks!

```suggestion
is modified after creation using ``ModifyNetworkInterfaceAttribute`` to specify this
```

Reply:
nice, updated!